### DISCUSSION TITLE
Watchlists not working for anyone but my own account

### DISCUSSION THREAD

**Discussion by Daxiongmao87** [sct/overseerr#3521] [Help]
Hey all, for some reason, Overseerr is doing absolutely nothing for other users with regards to their Watchlists.  They aren't imported, nothing in the logs indicate that they are even being checked for anyone besides my own (user_id: 1).  I've tried:

-  logging in to overseerr on a separate account
- disabled and re-enabled their permissions for auto-request and request
- ran the jobs multiple times, 
- tried restarting overseerr, 
- reconnecting plex, etc.

It used to work previously, but now it seems to only want to import just my (user_id: 1) watchlist.  

What am I missing?
![image](https://github.com/sct/overseerr/assets/967432/0bd5bb11-5768-4498-9b55-de0b9cb34a93)

Edit: The version of overseerr I'm using is 1.33.0

**Comment by WayneGibbings** [comment]
+1 same issue

**Comment by cl00dz** [comment]
i also have been having this issue.. they have the correct perms and everything but still seem to have an issue with my users watchlists not syncing with overseerr

**Comment by Anqui3tas** [comment]
Recently also noticed this is occurring with my instance.

**Comment by feline51** [comment]
Hello, same issue, I hope the next version will fix it :)

**Comment by helloimraj** [comment]
same issue as of 1.33.2

**Comment by poltergaiist** [comment]
Hi,
I had same issue, I removed the docker and re-added it and all watchlists just showed up for the other users. 
Hope it works for you as well
/BR

**Comment by philipjphry** [comment]
From what I can tell, each user has to login once to Overseerr to set up the Plex token.  After that, their watchlists should start syncing.  Has anyone been able to get it work without each user logging in separately?  I am not sure how I can get all my users to do so.
